spec/project_configuration_spec.rb

```diff
@@ -109,6 +109,28 @@
     end
   end
 
+  describe '#path' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.